.eslintignore

```diff
@@ -1 +1,3 @@
 coverage
+node_modules
+support
```

Comment:
We need to sort this out; anyway this should not be solved as this PR; lets move it out of the scope of this PR for now; @bajtos Any idea about this? I can take care of this after hearing your inputs...